App/Seller/Sources/Seller/UI/MessagesView/FilterSearchView.swift

```diff
@@ -0,0 +1,136 @@
+// system
+import SwiftUI
+
+struct FilterSearchView: View {
+    
+    @ObservedObject var viewModel: MyMessagesViewModel
```

Comment:
Please use `Observation` for this :)

https://developer.apple.com/documentation/swiftui/migrating-from-the-observable-object-protocol-to-the-observable-macro

Reply:
Migrated